repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql

```diff
@@ -0,0 +1,6 @@
+DROP INDEX idx_reference  CASCADE;
```

Comment:
Could we please use `IF EXISTS` and without `CASCADE` for all of these e.g.:
```suggestion
DROP INDEX IF EXISTS idx_reference;
```

Reply:
SQL Statements update, by removing CASCADE and adding IF EXISTS